include/tvm/topi/transform.h

```diff
@@ -1693,6 +1694,21 @@ inline Tensor auto_scheduler_layout_transform(const Tensor& src, const String& s
       name, tag);
 }
 
+/*!
+ * \brief Transform the meta-schedule generated layout according to TIR's IndexMap
+ * \param src the source input.
+ * \param index_map The TIR IndexMap
+ * \param name output tensor name.
+ * \param tag output tensor tag.
+ * \return A tensor. The layout transformation method
+ */
+inline Tensor meta_schedule_layout_transform(const Tensor& src, const tir::IndexMap& index_map,
+                                             const String name = "T_meta_schedule_layout_trans",
+                                             const String tag = kInjective) {
+  // TODO
```

Comment:
@jinhongyii I will need help on this

Reply:
Doesn't InverseAffineIterMap work?

Reply:
To this end, I was actually examining the code here but didn't found `InverseAffineIterMap` used anywhere in TransformLayout: https://github.com/junrushao1994/tvm/blob/65071978d01af5ba2ca9721ba51125fe46d6887d/src/tir/schedule/primitive/layout_transformation.cc#L135. am a bit confused now

Reply:
why do you expect InverseAffineIterMap to be used in layout_transformation?